.github/workflows/check-toc.yaml

```diff
@@ -11,11 +11,13 @@ jobs:
 
       - name: Verify ToC
         run: |
-          npx markdown-toc README.md -i
+          find . -name "*.md" | while read -r file; do
+              npx markdown-toc $file -i
+          done
           if [[ $(git status --porcelain) ]]; then
-              echo ERROR: "Did you forget to update the README table of contents?" >&2
-              echo ERROR: "Please run 'npx markdown-toc README.md -i' and update your PR" >&2
+              echo ERROR: "Did you forget to update markdowns table of contents?" >&2
```

Comment:
```suggestion
              echo ERROR: "Did you forget to update markdowns tables of contents?" >&2
```

Reply:
Was thinking about it yesterday. Not sure what is the proper plural form here.

Reply:
Whatever you decide, just make sure it is consistent with https://github.com/konflux-ci/konflux-ci/blob/b05d020eb02cc8858b1eb75749c5c1196da98404/.github/workflows/check-toc.yaml#L23